sql/core/src/test/resources/sql-tests/results/group-by.sql.out

```diff
@@ -1007,7 +1007,7 @@ SELECT
   percentile_disc(0.25) WITHIN GROUP (ORDER BY v DESC)
 FROM aggr
 -- !query schema
-struct<percentile_disc(v, 0.25, 1):double,percentile_disc(v, 0.25, 1):double>
+struct<percentile_disc(v, 0.25):double,percentile_disc(v, 0.25):double>
```

Comment:
I think this indicates that we need override `Percentile.sql`, otherwise we can't distinguish `ORDER BY v DESC` and `ORDER BY v ASC`